### PR TITLE
Acrolinx quality improvements: slb-gateway-performance.md

### PR DESCRIPTION
**Pull request by gswashington**

### CODE REVIEW

**Comment by Court72** [comment]
@robinharwood, @Xelu86

Can you review the proposed changes? *IMPORTANT*: When the changes are ready for publication, add a `#sign-off` comment to signal that the PR is ready for the review team to merge. 

#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

File: WindowsServerDocs/administration/performance-tuning/subsystem/software-defined-networking/slb-gateway-performance.md [hunk 1]
```diff
@@ -4,34 +4,38 @@ description: SLB Gateway performance tuning guidelines on SDN networks
...
 
 # SLB Gateway Performance Tuning in Software Defined Networks
 
-Software load balancing is provided by a combination of a load balancer manager in the Network Controller VMs, the Hyper-V Virtual Switch and a set of Load Balancer Multixplexor (Mux) VMs.
+Applies to: Windows Server (All supported versions) 
 
-No additional performance tuning is required to configure the Network Controller or the Hyper-V host for load balancing beyond what is described in the [Software Defined Networking](index.md) section, unless you will be using SR-IOV for the Muxes as described below.
+Software load balancing is provided by a combination of a load balancer manager in the Network Controller VMs, the Hyper-V Virtual Switch, and a set of Load Balancer Multixplexor (Mux) VMs.
+
+No additional performance tuning is required to configure the Network Controller or the Hyper-V host for load balancing beyond what is described in the [Software Defined Networking](index.md) section, unless you'll be using SR-IOV for the Muxes, as described below.
```
Comment (robinharwood) [inline WindowsServerDocs/administration/performance-tuning/subsystem/software-defined-networking/slb-gateway-performance.md]: Both these sentences are quite complex. Could they be simplified?

File: WindowsServerDocs/administration/performance-tuning/subsystem/software-defined-networking/slb-gateway-performance.md [hunk 2]
```diff
@@ -4,34 +4,38 @@ description: SLB Gateway performance tuning guidelines on SDN networks
...
+An individual connection to a Virtual IP (VIP) will always be sent to the same Mux, assuming the number of muxes remains constant and its throughput is limited to the throughput of a single Mux VM.  Muxes only process the inbound traffic that is going to a VIP.  Response packets go directly from the VM that is sending the response to the physical switch that forwards it on to the client.
 
-In some cases when the source of the request originates from an SDN host that is added to the same Network Controller that manages the VIP, further optimization of the inbound path for the request is also performed which enables most packets to travel directly from the client to the server, bypassing the Mux VM entirely.  No additional configuration is required for this optimization to take place.
+In some cases, when the source of the request originates from an SDN host that is added to the same Network Controller that manages the VIP, further optimization of the inbound path for the request is also performed. This enables most packets to travel directly from the client to the server, bypassing the Mux VM entirely. No additional configuration is required for this optimization to happen.
```
Comment (robinharwood) [inline WindowsServerDocs/administration/performance-tuning/subsystem/software-defined-networking/slb-gateway-performance.md]: Still quite complex, could it be split up or simplified?

File: WindowsServerDocs/administration/performance-tuning/subsystem/software-defined-networking/slb-gateway-performance.md [hunk 3]
```diff
@@ -4,34 +4,38 @@ description: SLB Gateway performance tuning guidelines on SDN networks
...
 
 Each SLB Mux VM must be sized according to the guidelines provided in the SDN infrastructure virtual machine role requirements section of the  [Plan a Software Defined Network Infrastructure](/azure-stack/hci/concepts/plan-software-defined-networking-infrastructure) topic.
```
Comment (robinharwood) [inline WindowsServerDocs/administration/performance-tuning/subsystem/software-defined-networking/slb-gateway-performance.md]: Also quite complex. Could it be simplified?

**Review by robinharwood — CHANGES_REQUESTED** [review]
Hey @gswashington, just a few complex sentences to simplify.

Thanks,
Robin

**Comment by robinharwood** [comment]
@gswashington, a few updates but looks good. Merging now.

**Pull request merged**